specs/agents/tracing-instrumentation-messaging.md

```diff
@@ -188,6 +197,25 @@ queues/topics/exchanges will be ignored.
 | Central config | `true` |
 
 
+### Message metadata
+
+To support distributed tracing with automatic instrumentation, the messaging
+system must provide a mechanism to add metadata/properties/attributes to
+individual messages, akin to HTTP headers. If an APM agent supports
+trace-context for a given messaging system, it MUST use the following mechanisms
+so that cross-language tracing works:
+
+| Messaging system       | Mechanism |
+| ---------------------- | --------- |
+| Azure Queue            | Unknown |
+| Azure Service Bus      | Unknown |
```

Comment:
For Azure Service Bus it would be this one, I guess: https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-end-to-end-tracing?tabs=net-standard-sdk-2

Reply:
Thanks. I added a note in https://github.com/elastic/apm/pull/616/commits/6f73489626bbf8d039e9b9c6f345625e14b25005